tools/distpartitioning/convert_partition.py

```diff
@@ -175,109 +175,61 @@ def _get_unique_invidx(srcids, dstids, nids):
-       "ntype-name" is the user assigned name to this node type. "format" describes the
-       format of the contents of the files. and "data" is a list of lists, each list has
-       3 elements: file-name, start_id and end_id. File-name can be either absolute or
-       relative path to this file and starting and ending ids are type ids of the nodes
-       which are contained in this file. These type ids are later used to compute global ids
-       of these nodes which are used throughout the processing of this pipeline.
-        "ntype-name" : {
-            "format" : "csv",
-            "data" : [
-                    [ <path-to-file>/ntype0-name-0.csv, start_id0, end_id0],
-                    [ <path-to-file>/ntype0-name-1.csv, start_id1, end_id1],
-                    ...
-                    [ <path-to-file>/ntype0-name-<p-1>.csv, start_id<p-1>, end_id<p-1>],
-            ]
-        }
-
-    2. The key-value pairs in the "eid" dictionary has the following format.
-       As described for the "nid" dictionary the "eid" dictionary is similarly structured
-       except that these entries are for edges.
-        "etype-name" : {
-            "format" : "csv",
-            "data" : [
-                    [ <path-to-file>/etype0-name-0, start_id0, end_id0],
-                    [ <path-to-file>/etype0-name-1 start_id1, end_id1],
-                    ...
-                    [ <path-to-file>/etype0-name-1 start_id<p-1>, end_id<p-1>]
-            ]
-        }
-
-    In "nid" dictionary, the type_nids are specified that
-    should be assigned to nodes which are read from the corresponding nodes file.
-    Along the same lines dictionary for the key "eid" is used for edges in the
-    input graph.
-
-    These type ids, for nodes and edges, are used to compute global ids for nodes
-    and edges which are stored in the graph object.
+    This is an auxiliary function used to process data relating to the nodes
+    of the current partition and prepares the data needed to create the dgl
+    graph object. The data accumulated from the beginning of the pipeline
+    after this processing step will be deleted to free up the memory footprint
+    of the pipeline.
 
     Parameters:
     -----------
-    schame : json object
-        json object created by reading the graph metadata json file
-    part_id : int
-        partition id of the graph partition for which dgl object is to be created
-    node_data : numpy ndarray
-        node_data, where each row is of the following format:
-        <global_nid> <ntype_id> <global_type_nid>
-    edge_data : numpy ndarray
-        edge_data, where each row is of the following format:
-        <global_src_id> <global_dst_id> <etype_id> <global_type_eid>
-    edgeid_offset : int
-        offset to be used when assigning edge global ids in the current partition
-    return_orig_ids : bool, optional
-        Indicates whether to return original node/edge IDs.
+    schema : json dictionary
```

Comment:
Can you follow the DGL convention, e.g.
https://github.com/dmlc/dgl/blob/f5ddb11448335c45b16fb1d0561c490d937ab8f5/python/dgl/data/graph_serialize.py#L100

Reply:
Same for others.

Reply:
and other types, such as numpy array -> numpy.array